feature_engine/transformation/boxcox.py

```diff
@@ -138,6 +139,30 @@ def transform(self, X: pd.DataFrame) -> pd.DataFrame:
 
         return X
 
+    def inverse_transform(self, X: pd.DataFrame) -> pd.DataFrame:
+        """
+        Apply the inverse BoxCox transformation.
```

Comment:
Here, for convention we write the following:

Convert the data back to the original representation.

Instead of what you wrote, that makes total sense. It's just that we want to follow scikit-learn conventions.

Reply:
Sorry, I forgot to update the class docs, I've did the changes now.
Thanks.